src/styles/00-base/02-typography/01-typefaces/_typefaces.scss

```diff
@@ -1,5 +1,5 @@
 // Typography Variables
-@import '../../node_modules/system-font-css/system-font.scss';
+@import '~system-font-css/_system-font.scss';
```

Comment:
This has an underscore from the npm package? That's new, I feel like I haven't seen that before

Reply:
I'm not sure why it isn't showing up in github's source, but when you npm i and look under node_modules, it shows up as `_system-font.scss`.  I wonder though if you're supposed to import `system-font.css` which is the compiled version ...